### DISCUSSION TITLE
Add BeautifulSoup4 Package

### DISCUSSION THREAD

**Discussion by ewwink** [cloudflare/workerd#1956] [Python Packages]
Add BeautifulSoup4 Package

**Comment by clintgibler** [comment]
Adding support for BeautifulSoup4 or some way to easily parse HTML would be huge 🙏

**Comment by devsoft-co-za** [comment]
This would help me also.

**Comment by soorower** [comment]
Upvote!!

Also, lxml for web scraping (super fast than bs4)

**Comment by McSpidey** [comment]
Another upvote for this

**Comment by li0ard** [comment]
Yet another upvote

**Comment by li0ard** [comment]
We still waiting

**Comment by VityaSchel** [comment]
@li0ard asked me to upvote this so here i am

**Comment by amovsesy** [comment]
Upvote

**Comment by ryanking13** [comment]
BeautifulSoup4 works in Python workers now.
